help/text-emphasis.md

```diff
@@ -0,0 +1,45 @@
+# Text emphasis
+
+Zulip supports Markdown syntax for bold, italic, and strikethrough text.
+
+{!emphasis.md!}
+
+## Using the bold text button
+
+{start_tabs}
+
+{!start-composing.md!}
+
+1. Select the text you want to format.
+
+1. Click the **bold text** (<i class="fa fa-bold"></i>) icon at the
+   bottom of the compose box.
+
+!!! keyboard_tip ""
+
+    You can also use <kbd>Ctrl</kbd> + <kbd>B</kbd> to insert bold text.
```

Comment:
Maybe "... to insert bold formatting."? You aren't really inserting text, so I think the current wording is a bit confusing, but open to other options.

Reply:
Yeah, that makes sense. I was going with the wording we used in [keyboard shortcuts](https://zulip.com/help/keyboard-shortcuts#in-the-compose-box), so I guess we should improve the wording there too.